SECURITY.md

```diff
@@ -0,0 +1,67 @@
+# Security Policy
+
+Being a security tool itself, the security of Suricata itself is naturally of
+great importance. This document describes the policy around security issues.
+
+
+## Severity Levels
+
+We will determine the severity of each issue, taking into account our
+experience dealing with past issues, versions affected, common defaults,
+and use cases. We use the following severity categories:
+
+* **CRITICAL** Severity. This affects Tier 1 features that are enabled by default
+where the issue disrupts availability of the service, leading to severe
+loss of visibility and/or availability. RCE, remotely triggerable traffic
```

Comment:
Should we remove RCE if we describe more precisely remotely triggerable traffic based crashes ?

Reply:
I feel it's a separate class of issues that we can mention separately here.

Reply:
not sure, seems more specific than we need to be?

Reply:
So, I do not understand what is meant here by "RCE".

Is arbitrary file write with datasets rule RCE to you ?

Anyways, I think this deserves something more precise than an acronym (and one that is notoriously vague)